frontend/src/slices/projects/projects.slice.ts

```diff
@@ -21,6 +21,16 @@ const { reducer, actions, name } = createSlice({
   name: 'projects',
   reducers: {},
   extraReducers(builder) {
+    builder.addCase(createProject.pending, (state) => {
+      state.dataStatus = DataStatus.PENDING;
+    });
+    builder.addCase(createProject.fulfilled, (state, action) => {
+      state.dataStatus = DataStatus.FULFILLED;
+      state.projects = [...state.projects, action.payload];
```

Comment:
Probably would be better if you create projects array in createProject thunk
```suggestion
      state.projects = action.payload;
```

Reply:
@dimapopovych Why? I find it very strange if 'createProject thunk' returns all projects since it only creates one

Reply:
no, we need to add a new project to the end of the array. so everything is fine

Reply:
I mean something like this
![image](https://user-images.githubusercontent.com/100519494/228188737-5a4453fd-c087-44e7-b70f-fc8c8f06d579.png)
But I'm not sure if it's good approach